scss/_patterns_lists.scss

```diff
@@ -199,6 +216,49 @@ $list-step-bullet-margin: $sph--x-large;
   }
 }
 
+@mixin vf-p-list-horizontal-section {
+  .p-list--horizontal-section {
```

Comment:
Because of how different it is from standard "lists", I wonder if making it a "variant" of `p-list` is justified, or should it be more like it's own component (like `p-horizontal-list-section` or something).

On the other hand, keeping it as `p-list--` we can reuse existing `p-list__item` and bullet/ticked options, so I guess there is a value in that consistency.

Reply:
Yes, this component behaves a bit like an HOC in that it is a rather featured combination of grid and list. However, it requires its own styling. So, I've implemented it as a list variant to be able to take advantage of other list classes and internal placeholders like you have mentioned.